tl

```diff
@@ -18,10 +18,21 @@ local function die(msg)
    os.exit(1)
 end
 
+local function find_in_sequence(seq, value)
```

Comment:
The PR looks great but the indentation is a bit off? (4 spaces in a few places)

Reply:
My bad! Should be fixed, I think.